cypress/support/fragments/settings/notes/noteTypes.js

```diff
@@ -52,13 +56,35 @@ export default {
     cy.expect(saveNoteTypeButton.has({ disabled: false }));
   },
 
-  saveNoteType: (noteType) => {
+  saveNoteType(noteType) {
     cy.do(saveNoteTypeButton.click());
     // need to wait for note type to appear after creation
     cy.wait(2000);
-    cy.expect(MultiColumnListRow({ content: including(noteType) }).exists());
+    this.checkNoteTypeIsDisplayed(noteType);
+  },
+
+  deleteNoteType(noteType) {
+    this.clickDeleteNoteType(noteType);
+    ConfirmDelete.verifyDeleteMessage(noteType);
+    ConfirmDelete.confirmDelete();
   },
 
+  clickEditNoteType: (noteType) => cy.do(
+    MultiColumnListRow({ content: including(noteType) })
```

Comment:
Please make this element constant, as it is used several times in that file

Reply:
fixed